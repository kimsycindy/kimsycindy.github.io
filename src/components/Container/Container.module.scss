@import '../../styles/variables.scss';

.Container {
  position: relative;
  margin-left: auto;
  margin-right: auto;
  width: 100%;
  max-width: $width-lg;
}
