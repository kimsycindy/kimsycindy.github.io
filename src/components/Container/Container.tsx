import React from 'react'
import cx from 'classnames'

import styles from './Container.module.scss'

interface Props {
  className?: string
}

const Container: React.FC<Props> = ({ children, className }) => (
  <div className={cx(styles.Container, className)}>{children}</div>
)

export default Container
