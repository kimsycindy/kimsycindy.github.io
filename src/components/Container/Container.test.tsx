import React from 'react'
import renderer from 'react-test-renderer'

import Container from './Container'

describe('Container', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Container className="container-classname">
          <p>Hello world!</p>
        </Container>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
