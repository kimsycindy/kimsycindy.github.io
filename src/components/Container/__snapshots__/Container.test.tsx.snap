// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container renders correctly 1`] = `
<div
  className="Container container-classname"
>
  <p>
    Hello world!
  </p>
</div>
`;
